test/csuite/wt2853_perf/main.c

```diff
@@ -86,6 +86,10 @@ main(int argc, char *argv[])
 
 	opts = &_opts;
 	sharedopts = &_sharedopts;
+
+	/* Check if the WT_DISABLE_LONG_TESTS is set and if so, exit */
```

Comment:
I don't think we need a comment here - the name of the function makes the functionality obvious.